Resources/Textures/Objects/Weapons/Guns/Projectiles/flesh_toxic.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "license": "CC-BY-NC-3.0",
```

Comment:
any reason for it being `CC-BY-NC-3.0` and not `CC-BY-SA-3.0` as all other RSIs in this PR?

Reply:
i dont know, maybe typo